webpack.prod.js

```diff
@@ -12,6 +13,14 @@ module.exports = merge(common, {
       new webpack.DefinePlugin({
          WEBPACK_MODE: JSON.stringify("production"),
          VERSION: JSON.stringify(process.env.npm_package_version),
+         SENTRY_DSN: JSON.stringify(
+            "https://c16443e39a66eae141954dfd23890812@o144358.ingest.sentry.io/4505832903147520"
```

Comment:
is this something we would also want to put in our `process.env.SENTRY_DSN` settings?  kinda like `SENTRY_AUTH_TOKEN` ?

Don't really know, just asking.

Reply:
> DSNs are safe to keep public because they only allow submission of new events and related event data; they do not allow read access to any information. (https://docs.sentry.io/product/sentry-basics/dsn-explainer/)

This get's injected into the code during the webpack build. We could put into an env variable, but I don't think that's necessary.

The `SENTRY_AUTH_TOKEN` is different. It is used to log into sentry and upload the generated source maps for new versions. This happens in our CI and we have the secret set in the `env` there.

Reply:
I was thinking more about keeping things easily configurable.  So for example if another group wanted to use our framework, is this a setting they would need to change for their setup?  If so, then I would rather keep all those configurable settings in a common place like the `.env` file.

Reply:
It difficult because these get baked into the web code, so they would need to build a custom web image anyways. Which includes checking out two+ repos and running webpack. I'm not sure that using an `.env` file now makes a meaningful difference for easy configuration. All that to say, I think this would be part of a bigger task on making the `ab_web` image configurable. Might also include how to add plugins, custom assets, etc.